java/vector/src/main/codegen/templates/UnionListWriter.java

```diff
@@ -231,12 +234,31 @@ public void endList() {
     listStarted = false;
   }
 
+  @Override
   public void startListView() {
     vector.startNewValue(idx());
     writer.setPosition(vector.getOffsetBuffer().getInt((idx()) * OFFSET_WIDTH));
     listStarted = true;
   }
 
+  @Override
+  public void endListView() {
+    int sizeUptoIdx = 0;
+    for (int i = 0; i < idx(); i++) {
+      sizeUptoIdx += vector.getSizeBuffer().getInt(i * SIZE_WIDTH);
+    }
+    vector.getSizeBuffer().setInt(idx() * SIZE_WIDTH, writer.idx() - sizeUptoIdx);
+    setPosition(idx() + 1);
+    listStarted = false;
+  }
+  <#elseif listName == "LargeListView">
+  @Override
+  public void startListView() {
+    vector.startNewValue(idx());
+    writer.setPosition(checkedCastToInt(vector.getOffsetBuffer().getInt((idx()) * OFFSET_WIDTH)));
+    listStarted = true;
+  }
+
```

Comment:
ListView includes `startList()` and `endList()` apis. Should LargeListView have them, too?

Reply:
Yeah probably we should. I will rectify that. Thanks for catching this.